CSS/index.html

```diff
@@ -0,0 +1,109 @@
+            <div class="filing-box">
+                <label for="name" class="title">Your company name<span>*</span></label>
+                <input required id="name" type="text" name="ame" class="field" />
+            </div>
+            <div class="filing-box">
+                <label for="nature" class="title">Nature of Business<span>*</span></label>
+                <input required id="nature" type="text" name="nature" class="field" />
+            </div>
+            <div class="filing-box">
+                <label for="date" class="title">Date of registration</label>
+                <input id="date" name="date" placeholder="DD/MM/YYYY" pattern="^(0[1-9]|1[0-2])\/([0-9]{2})$" class="field" />
+            </div>
+            <div class="filing-box">
+                <label for="country" class="title">Country of registartion<span>*</span></label>
+                <select id="country" name="country" required class="field">
+                    <option value=""disabled selected hidden>Select</option>
+                    <option value="Ukraine">Ukraine</option>
+                    <option value="China">China</option>
+                    <option value="UK">UK</option>
+                </select>
+            </div>
+            <div class="filing-box">
+                <label for="adress" class="title">Adress</label>
+                <input id="adress" type="text" name="adress" class="field" />
+            </div>
+            <div class="filing-box">
+                <label for="postcode" class="title">Postcode</label>
+                <input id="postcode" type="text" pattern="[A-Z]{2}[0-9]{2}" name="postcode" class="field" />
+            </div>
+       </div>
+       <div class="personal-data">
+            <div class="filing-box">
+                <label for="contactName" class="title">Contact name</label>
+                <input id="contactName" type="text" name="contactName" class="field" />
+            </div>
+            <div class="filing-box">
+                <label for="contactPhone" class="title">Contact phone</label>
+                <input id="contactPhone" type="tel" pattern="[0-9]{3}-[0-9]{3}" name="contactPhone" class="field" />
+            </div>
+            <div class="filing-box">
+                <label for="email" class="title">Email<span>*</span></label>
+                <input required id="email" type="email" name="email" class="field" />
+            </div>
+            <div class="filing-box">
+                <label for="idea" class="title">Let's talk about your idea</label>
+                <textarea id="idea" name="idea" maxlength="320" class="field big-field"></textarea>
+            </div>
+            <span class="multi-range">
+                <input type="range" min="0" max="100" value="0" id="lower" />
+                <input type="range" min="0" max="100" value="50" id="upper" />
+            </span>
+        </div>
+        <div class="agreement">
+                <label class="dashed-box">
```

Comment:
Not the best name. This container is not only dashed-box. It is a label for file upload